style.css

```diff
@@ -0,0 +1,358 @@
+  text-align: left;
+  padding-left: 3%;
+  padding-top: 5%;
+  margin-bottom: unset;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.grid-container > div > div {
+  display: flex;
+  flex-direction: row;
+}
+
+.grid-container > div > div > ul {
+  margin-bottom: unset;
+  padding-left: 4%;
+  margin-top: unset;
+}
+
+.about_section > p {
+  padding-left: 5%;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.grid-container > div > p {
+  text-align: left;
+  padding-left: 3%;
+  padding-top: 4%;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.diff_classes {
+  margin-top: unset;
+  margin-left: unset;
+  width: 210px;
+}
+
+.about_section > h3 {
+  width: 40%;
+  padding: 4%;
+  padding-left: unset;
+  padding-right: unset;
+  margin-left: 3%;
+  color: #396df2;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border-radius: 10px;
+  text-align: center;
+  border: 1px solid blue;
+}
+
+.grid-container > div > h3 {
+  width: 40%;
+  padding: 4%;
+  padding-left: unset;
+  padding-right: unset;
+  margin-left: 3%;
+  color: #396df2;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border-radius: 10px;
+  text-align: center;
+  border: 1px solid blue;
+}
+
+.grid-item {
+  background-color: rgba(255, 255, 255, 0.8);
+  text-align: center;
+  padding: 10px;
+  font-size: 30px;
+  border-radius: 15px;
+  border: 1px solid lightgray;
+}
+
+.item1 {
+  grid-column: 1 / span 2;
+  grid-row: 1;
+}
+
+.item2 {
+  grid-column: 1 / span 2;
+  grid-row: 2;
+}
+
+.item3 {
+  grid-column: 1 / span 2;
+  grid-row: 3;
+}
+
+.item4 {
+  grid-column: 1 / span 2;
+  grid-row: 4;
+}
+
+.about_section {
+  display: flex;
+  flex-direction: column;
+  margin-top: 20%;
+  width: 400px;
+  border-radius: 10px;
+  background-color: white;
+}
+
+.about_section > ul > li:first-child {
+  padding-left: unset;
+}
+
+.lang_section {
+  display: flex;
+  background-color: white;
+  padding-left: 7%;
+  width: 360px;
+}
+
+.lang_section > h2 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  padding-left: 3%;
+}
+
+.lang_section > i {
+  font-size: 30px;
+  padding-top: 4.4%;
+  padding-left: 35%;
+}
+
+.js_flex,
+.html_flex,
+.css_flex {
+  display: flex;
+  background-color: #f7f7f9;
+  padding-left: 2%;
+  width: 60%;
+  margin: 4%;
+  margin-left: 11%;
+}
+
+.framework,
+.skill_div {
+  display: flex;
+  background-color: white;
+}
+
+.framework > i {
+  padding-top: 7%;
+  font-size: 23px;
+  padding-left: 40%;
+}
+
+.skill_div > i {
+  padding-top: 7%;
+  font-size: 23px;
```

Comment:
[OPTIONAL]
**Don't Repeat Yourself**
- Keep your code optimized and reduce same lines of code.

Reply:
This is optional comment. Will follow this in next project

Reply:
Nice to know that. I hope that this [article](https://medium.com/circuits-technical-talks/dry-css-9cbe3215af3d) it's gonna be helpful for you. Read it once that you have a chance.